pages/common/gixy.md

```diff
@@ -0,0 +1,20 @@
+# gixy
+
+> Analyze Nginx configuration.
+> More information: <https://github.com/yandex/gixy>.
+
+- Analyze nginx configuration (default path: /etc/nginx/nginx.conf):
```

Comment:
Should Nginx be stylised the same everywhere (i.e. Nginx, NGINX, nginx) for consistency?